### PR TITLE
[FLINK-20967][docs] HBase properties 'properties.*' function Description

### PR DESCRIPTION
**Pull request by chaozwn**
## What is the purpose of the change

add change doc.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Review by wuchong — CHANGES_REQUESTED** [review]
Why adding new pages instead of updating the existing one? `docs/dev/table/connectors/hbase.md`

**Comment by chaozwn** [comment]
> Why adding new pages instead of updating the existing one? `docs/dev/table/connectors/hbase.md`

sorry,because the first pr I submitted was not copied from the master branch, and the previous commit was mixed, I copied the two changed files and resubmitted a new pr. It doesn't matter, the two documents are actually the same. I will revise it tomorrow.

**Comment by wuchong** [comment]
Thanks for the explanation and working on this.

**Comment by chaozwn** [comment]
> Thanks for the explanation and working on this @chaozwn.

I have modified the corresponding document, please help to check whether the description is accurate.

File: docs/dev/table/connectors/hbase.md [hunk 1]
```diff
@@ -172,6 +172,15 @@ Connector Options
       <td>Integer</td>
       <td>Defines the parallelism of the HBase sink operator. By default, the parallelism is determined by the framework using the same parallelism of the upstream chained operator.</td>
     </tr>
+    <tr>
+      <td><h5>properties.*</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>
+         This can set and pass arbitrary HBase configurations. Suffix names must match the configuration key defined in <a href="https://abloz.com/hbase/book.html#perf.configurations">HBase Configuration documentation</a>. Flink will remove the "properties." key prefix and pass the transformed key and values to the underlying HBaseClient. For example, you can add a kerberos authentication parameter <code>'properties.hbase.security.authentication' = 'kerberos'</code>.
```
Comment (wuchong) [inline docs/dev/table/connectors/hbase.md:181]: Please use an official HBase link, e.g. http://hbase.apache.org/2.3/book.html#hbase_default_configurations ?

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**